.travis.yml

```diff
@@ -13,7 +13,11 @@ addons:
       - lcov
 compiler:
   - gcc
+env:
+  - MATRIX_EVAL="CC=gcc-7 && CXX=g++-7"
```

Comment:
why not just:

```yml
env:
  - CC=gcc-7 CXX=g++-7
```

then you can skip the `eval`

Reply:
That’s true, I was just trying to avoid changing things *too* much. More relevantly though, do you think we would want to get it building on clang *and* gcc? Then having a separate matrix would make sense (I could try to set that up when I get home in ~ half an hour).

Reply:
It would be nice down the line, but based on the troubles we've had so far it might be timeconsuming to keep everything working with both at the same time, but it's up to you

Reply:
I tried removing it and it breaks the build 🤷‍♀️ For some reason it seems like if you just put the environment variables directly it doesn't pick them up and reverts to the (pretty old) gcc compiler built in.

Reply:
Interesting 🤔 oh well